### ISSUE TITLE
The options for making a backup are (or should be) separate options, thus one should not disable the other

### ISSUE THREAD

**Issue by Meteor0id** [signalapp/Signal-Android#8493]
- [x ] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x ] This isn't a feature request or a discussion topic

----------------------------------------

I was going though the translations and noticed an option which wasn't clear to me. Why is there an option "Create backup" when there is already an option "Chat backups"?
I have seen many issues like these where the original English text string is ambiguous. Normally I would raise an issue in Transifex. In this case however, the functionality of the options also makes no sense.

There are two options:
"Chat backups" which creates backups automatically at a time interval.
"Create backup" which creates a backup right at this moment, and will not automatically repeat unless the other option is enabled.

Now look at the screenshot. Does that seem obvious? It doesn't to me. In fact it is confusing.

Even worse, Why can't I create a backup right now, without enabling automated backups? Why is the second option disabled, when the first option is switched off?

I don't think the backup functions where supposed to be implemented like this, and if they were, I still suggest changing them.
I propose the following:

Change the text strings:

> 
> "Create backups automatically"
> "Backup all conversations to external storage every now and then"

> 
> "Create a backup now"
> "Last backup: Never"

The first option should not disable the second. One should not depend on the other. At enabling either one, check if the required permissions were given.

Maybe we should also add an option:

> 
> "Reset the password for backups"
> "Stop making backups with the current password, and create a new backup file with a new password. You must do this if you lost your 30-digit your password."

Don't forget to update text stings for the support center.

![screenshot signal backups 1](https://user-images.githubusercontent.com/34976212/50459913-4deee600-0970-11e9-9c96-747f60b2c58f.png)

**Comment by JasonMFry** [comment]
This seems pretty straightforward, but I'm new to Java, Android Studio, and this codebase so if someone could guide me a bit I'd be more than happy to help.

I've found `res/values/strings.xml`, but that doesn't help much :) I've got to sign off for now but I think I need to be in `src/org/thoughtcrime/securesms/backup/`. If someone could at least confirm, and maybe guide me a little more, that'd help.
